src/prog_models/models/aircraft_model/vehicles/vehicles.py

```diff
@@ -56,21 +56,24 @@ def TAROT18(payload=0.0, gravity=9.81):
                     Gamma=None,     # Thrust allocation matrix (constant, depending on kt, kq, and the UAV rotor configuration)
                     Gamma_inv=None,     # Inverse of thrust allocation matrix (constant, depending on kt, kq, and the UAV rotor configuration)
                     )
-    
+
     mass = rotorcraft_masses(mass, geom)   # compute total and body mass
     mass, geom = rotorcraft_inertia(mass, geom)   # Build rotorcraft inertia properties
     if payload > mass['max_payload']:
-        warn("Payload for TAROT 18 exceeds its maximum recommended payload.")
+        raise Warning("Payload for TAROT 18 exceeds its maximum recommended payload.")
```

Comment:
```suggestion
       warn("Payload for TAROT 18 exceeds its maximum recommended payload.")
```

We definitely want this to be warn. I suspect reversing this change was a merge issue